modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -75,6 +75,15 @@ export default {
     docId: {
       type: String,
       default: null
+    },
```

Comment:
@boutell @abea I decided to pass "unpublished" and "trash" filters to the request fetching data in the backend. This is to avoid a bizarre situation where every unpublished or trash status was allowed and tests were failing (or needed to be redesigned).